Makefile

```diff
@@ -58,6 +73,17 @@ build-controller.amd64:
 build-controller.arm64v8:
 	$(COMMONENVVAR) $(BUILDENVVAR) GOARCH=arm64 go build -ldflags '-w' -o bin/controller cmd/controller/controller.go
 
+.PHONY: build-controller.arm64
```

Comment:
The key takeaway of turning to distroless build is to use one single command to build the eventual distroless image, and hence we can eliminate these duplicates build entries for different platforms.

You can refer here to define [platforms](https://github.com/kubernetes-sigs/kueue/blob/f35494df6897ace0a9eed17443b4dac416467aaa/Makefile#L29) first, and then [run a single command](https://github.com/kubernetes-sigs/kueue/blob/f35494df6897ace0a9eed17443b4dac416467aaa/Makefile#L200-L207) to build the image.

PS: the underlying command is `docker buildx --platforms`.